### PR TITLE
fix smart-truncate

### PR DESCRIPTION
**Pull request by Grawl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/millerized/smart-truncate/
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/smart-truncate/smart-truncate-tests.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { smartTruncate } from 'smart-truncate';
+import smartTruncate from 'smart-truncate';
```
Comment (sandersn) [inline types/smart-truncate/smart-truncate-tests.ts:1]: Use the commonjs syntax `import smartTruncate = require('smart-truncate')`. Also please remove esModuleInterop from tsconfig.

Consumers of the package who want to use `import smartTruncate from 'smart-truncate'` will need to add `"esModuleInterop": true` as you did, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/smart-truncate/smart-truncate-tests.ts (hunk 1 above)
Comment (Grawl) [reply types/smart-truncate/smart-truncate-tests.ts:1]: if I change first line to this:

```ts
const smartTruncate = require('smart-truncate');
```

I will get this error in tests:

```
Error: /workspace/DefinitelyTyped/types/smart-truncate/smart-truncate-tests.ts:1:23
ERROR: 1:23  no-var-requires  require statement not part of an import statement
ERROR: 1:23  expect           TypeScript@4.4 compile error: 
Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node` and then add 'node' to the types field in your tsconfig.
```

Are you sure I have to write `import smartTruncate = require('smart-truncate')`?

Comment (Grawl) [reply types/smart-truncate/smart-truncate-tests.ts:1]: OK I see this syntax for the first time but found the same at [`types/slice-source/slice-source-tests.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/slice-source/slice-source-tests.ts)

**Review by sandersn — APPROVED** [review]

**Comment by Grawl** [comment]
Ready to merge

**Pull request merged**